### PR TITLE
Issue 593

### PR DESCRIPTION
**Pull request by kvarforl**

### CODE REVIEW

**Comment by saramsey** [comment]
OK this is great work. There may be a few merge conflicts when we finally merge kg2-curie-refactoring into master, but it probably won't be too bad.

**Comment by kvarforl** [comment]
Currently testing  on kg2dev and finding a couple of typos: wait a second to merge :)

**Comment by kvarforl** [comment]
tested by running build-kg2.sh test on kg2dev

File: code/kg2/extract-chembl.sh [hunk 1]
```diff
@@ -9,36 +9,36 @@ if [[ "${1:-}" == "--help" || "${1:-}" == "-h" ]]; then
...
-CHEMBL_DB_TARBALL=chembl_${CHEMBL_VERSION}_mysql.tar.gz
-CHEMBL_SQL_FILE=${CHEMBL_DIR}/chembl_${CHEMBL_VERSION}/chembl_${CHEMBL_VERSION}_mysql/chembl_${CHEMBL_VERSION}_mysql.dmp
+chembl_dir=${BUILD_DIR}/chembl
+chembl_version=25
+chembl_db_tarball=chembl_${chembl_version}_mysql.tar.gz
+chembl_sql_file=${chemble_dir}/chembl_${chembl_version}/chembl_${chembl_version}_mysql/chembl_${chembl_version}_mysql.dmp
 
-rm -r -f ${CHEMBL_DIR}
-mkdir -p ${CHEMBL_DIR}
+rm -r-f ${chemble_dir}
```
Comment (amykglen) [inline code/kg2/extract-chembl.sh:25]: ope - looks like a couple typos here?

- `-r -f` was changed to `-r-f`
- looks like you have an extra 'e' in '`chemble_dir`'

File: code/kg2/extract-chembl.sh [hunk 2]
```diff
@@ -9,36 +9,36 @@ if [[ "${1:-}" == "--help" || "${1:-}" == "-h" ]]; then
...
+mkdir -p ${chembe_dir}
```
Comment (amykglen) [inline code/kg2/extract-chembl.sh:26]: looks like a typo here as well (should be `chembl_dir`)

File: code/kg2/extract-ncbigene.sh [hunk 3]
```diff
@@ -12,17 +12,17 @@ fi
...
 date
 
-CONFIG_DIR=`dirname "$0"`
-source ${CONFIG_DIR}/master-config.shinc
+config_dir=`dirname "$0"`
+source ${config_dir}/master-config.shinc
 
-NCBI_GENE_TSV_FILE=${1:-"${BUILD_DIR}/ncbigene/Homo_sapiens_gene_info.tsv"}
-OUTPUT_DIR=`dirname ${NCBI_GENE_TSV_FILE}`
+ncbi_tsv_gene_file=${1:-"${BUILD_DIR}/ncbigene/Homo_sapiens_gene_info.tsv"}
```
Comment (amykglen) [inline code/kg2/extract-ncbigene.sh:18]: should this be `ncbi_gene_tsv_file`?

File: code/kg2/extract-chembl.sh [hunk 4]
```diff
@@ -9,36 +9,36 @@ if [[ "${1:-}" == "--help" || "${1:-}" == "-h" ]]; then
...
+source ${config_dir}/master-config.shinc
 
-CHEMBL_DIR=${BUILD_DIR}/chembl
-CHEMBL_VERSION=25
-CHEMBL_DB_TARBALL=chembl_${CHEMBL_VERSION}_mysql.tar.gz
-CHEMBL_SQL_FILE=${CHEMBL_DIR}/chembl_${CHEMBL_VERSION}/chembl_${CHEMBL_VERSION}_mysql/chembl_${CHEMBL_VERSION}_mysql.dmp
+chembl_dir=${BUILD_DIR}/chembl
+chembl_version=25
+chembl_db_tarball=chembl_${chembl_version}_mysql.tar.gz
+chembl_sql_file=${chemble_dir}/chembl_${chembl_version}/chembl_${chembl_version}_mysql/chembl_${chembl_version}_mysql.dmp
```
Comment (amykglen) [inline code/kg2/extract-chembl.sh:23]: ope - looks like there's still one typo in this line ('`chemble_dir`')

File: code/kg2/extract-chembl.sh [hunk 5]
```diff
@@ -9,36 +9,36 @@ if [[ "${1:-}" == "--help" || "${1:-}" == "-h" ]]; then
...
 
-${CURL_GET} ftp://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/latest/${CHEMBL_DB_TARBALL} > ${CHEMBL_DIR}/${CHEMBL_DB_TARBALL}
+${CURL_GET} ftp://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/latest/${chembl_db_tarball} > ${chembl_dir}/${chembl_db_tarball}
 
-tar xzf ${CHEMBL_DIR}/${CHEMBL_DB_TARBALL} -C ${CHEMBL_DIR}
-rm -f ${CHEMBL_DIR}/${CHEMBL_DB_TARBALL}
-gzip ${CHEMBL_SQL_FILE}
+tar xzf ${chembl_dir}/${chembl_db_tarball} -C ${chembl_dir}
+rm -f ${chembl_dir}/${chembl_db_tarball}
+gzip {chembl_sql_file}
```
Comment (amykglen) [inline code/kg2/extract-chembl.sh:32]: looks like `{chembl_sql_file}` lost its leading `$` in this line

**Pull request merged**